### DISCUSSION TITLE
ExFAT Support

### DISCUSSION THREAD

**Discussion by ghost** [CosmosOS/Cosmos#3121] [Ideas]
Be Nice

**Comment by ADev531** [comment]
1. we don't have that much license fee
2. if it's that easy then make it yourselves

**Reply by ghost** [reply]
Ext4?

**Reply by ADev531** [reply]
Just make it yourselves
Dev team doesn't have time for file system

**Reply by Szymekk44** [reply]
No one here gets money or anything for their work, everyone does what they want to help. You can't expect 10 people to work for free because you demand it. Cosmos is open source, learn osdev and you will see its not that easy as "add 64 bit lol"
